suites/quincy/rbd/tier-2_rbd_mirror_regression.yaml

```diff
@@ -296,3 +296,30 @@ tests:
             imagesize: 2G
       polarion-id: CEPH-9511
       desc: Verify configuring mirror to same source and target should fail
+
+  - test:
+      name: Image shrink in primary cluster
+      module: test_rbd_mirror_shrink_image_primary.py
+      clusters:
+        ceph-rbd1:
+          config:
+            journal:
+              ec_pool_config:
+                size: 2G
+                io_total: 10
```

Comment:
I think you are not using this `io_total` config option we can remove it?

Reply:
I am using that for running IOs , I am not going  with default values for io_total.

Reply:
ok got it.